app/server/http-servers/html-server.js

```diff
@@ -0,0 +1,54 @@
+import http from 'http';
+import fs from 'fs';
+import path from 'path';
+import through from 'through2';
+
+const port = 8081;
+const noIndexTemplate = `<html><head></head><body><h1>Somebody has stolen our template!!!</h1></body></html>\n`;
+
+http.createServer( requestHandler )
+    .listen(port, errorHandler);
+
+function requestHandler(req, res) {
+
+    const template = getTemplate('index.html');
+
+    res.writeHead(200, {'Content-Type': 'text/html'});
```

Comment:
If the template doesn't exist, would make sense return an appropriate HTTP status code. It isn't required, but can be considered as a part of best practices in API development :)